src/sentry/monitors/models.py

```diff
@@ -176,7 +190,7 @@ class Monitor(Model):
     project_id = BoundedBigIntegerField(db_index=True)
     name = models.CharField(max_length=128)
     status = BoundedPositiveIntegerField(
-        default=MonitorStatus.ACTIVE, choices=MonitorStatus.as_choices()
+        default=ObjectStatus.ACTIVE, choices=ObjectStatus.as_choices()
```

Comment:
ah wait we probably need a migration here

Reply:
Already done in https://github.com/getsentry/sentry/pull/48230

Reply:
ah got it. ok so let's remove `get_status_display` then and we're good ot go